resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/kubernetes/Client.scala

```diff
@@ -64,6 +64,9 @@ private[spark] class Client(
     .map(_.split(","))
     .getOrElse(Array.empty[String])
 
+  // CPU settings
+  private val driverCpuCores = sparkConf.getOption("spark.driver.cores").getOrElse("1")
```

Comment:
Yeah, there's not a config like `org.apache.spark.internal.config.DRIVER_MEMORY`. IMO, we shouldn't change the spark core code less as far as possible before we merge to spark master branch, but change kubernetes scheduler will be better.